src/Stripe.net/Services/Invoices/InvoiceService.cs

```diff
@@ -23,38 +23,38 @@ public InvoiceService(string apiKey)
 
         public bool ExpandSubscription { get; set; }
 
-        public virtual Invoice Get(string invoiceId, RequestOptions requestOptions = null)
+        public virtual Invoice Create(InvoiceCreateOptions createOptions = null, RequestOptions requestOptions = null)
         {
             return Mapper<Invoice>.MapFromJson(
-                Requestor.GetString(
-                    this.ApplyAllParameters(null, $"{Urls.Invoices}/{invoiceId}", false),
+                Requestor.PostString(
+                    this.ApplyAllParameters(createOptions, Urls.Invoices, false),
```

Comment:
Flagging for later, can we find a way to have the URL be an a class method/instance instead while we do that work of porting to the new services logic?

Reply:
well noticing you did it for refund below